### DISCUSSION TITLE
Vercel down

### DISCUSSION THREAD

**Discussion by IRediTOTO** [vercel/vercel#5069] [General] [answered]
Hi, this error from me or Vercel ? 
![image](https://user-images.githubusercontent.com/15695252/90899120-a4385480-e3f1-11ea-870e-40aa30b7a803.png)

https://www.vercel-status.com/

**Comment by IRediTOTO** [comment]
I use `vercel --prod` but after some minutes it return `Error! An internal error occurred with Vercel.`

**Comment by styfle** [comment, marked as answer]
Hi @IRediTOTO 

Indeed, there were some intermittent errors a couple days ago as mentioned on https://www.vercel-status.com

You should no longer see any errors and everything should be operating nominally, thanks!
